src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +18,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
```

Comment:
Using CompletableFuture.allOf without handling potential exceptions from the individual futures could lead to unobserved exceptions if any of the futures completes exceptionally. Consider adding exceptionally or handle methods to properly deal with exceptions.

Reply:
done